### DISCUSSION TITLE
SKR Mini e3 V3.0 + Stock Ender 6 Screen wont work

### DISCUSSION THREAD

**Discussion by hub-bhc** [Desuuuu/klipper#124] [General]
So i just tried to Compile a new Firmware with the make command after setting everything up with make menuconfig and somehow there is an error when trying to use the STM32G0B1 Processor wich is the chip on the SKR Mini E3 V3. 

Please Help :/
![Screenshot 2022-06-11 022614](https://user-images.githubusercontent.com/107285396/173165266-0a71673d-91c2-4f8c-baf5-3277ad6f1dca.png)

**Comment by etefield** [comment]
Have the same problem here , are we doing something wrong ?

**Reply by ghost** [reply]
It's only compatable with skr mini e3 2.0 and previous that I find to work and compile properly. I'd recommend switching to that instead.

**Comment by EllaFoxo** [comment]
I would suggest checking out the dgus-display branch, and also grab the screen DWIN_SET from [here](https://github.com/Desuuuu/DGUSPrinterMenu/releases). Works like a charm on the new version

On the SKR Mini E3 V3.0, the pinout for the display is on the TFT pins. Had to figure that one out the hard way, expecting it to be on EXP1

**Comment by Desuuuu** [comment]
The problem should be solved in commit 140dca1f68a46e5ee86f1ce4f99a8bc39e12b342.

Please note, the `dgus-display` branch is not actively developed anymore. Have a look at tinyfluffs' comment for an alternative.
